tests/impl.cpp

```diff
@@ -11796,7 +11807,7 @@ SSE2NEONTestImpl::SSE2NEONTestImpl(void)
     mTestFloatPointer2 = (float *) platformAlignedAlloc(sizeof(__m128));
     mTestIntPointer1 = (int32_t *) platformAlignedAlloc(sizeof(__m128i));
     mTestIntPointer2 = (int32_t *) platformAlignedAlloc(sizeof(__m128i));
-    srand(0);
+    x = 123456;
```

Comment:
Avoid such risky operation, which might be removed by other developers accidentally. Introduce a helper function instead.